include/manif/impl/se3/SE3Tangent_base.h

```diff
@@ -198,12 +198,18 @@ SE3TangentBase<_Derived>::rjac() const
   }
 
   /// @note Barfoot14tro Eq. 102
+  const Eigen::Matrix<Scalar, 3, 3> VW  = V * W;
+  const Eigen::Matrix<Scalar, 3, 3> WV  = VW.transpose();       // Note on this change wrt. Barfoot: it happens that V*W = (W*V).transpose() !!!
+  const Eigen::Matrix<Scalar, 3, 3> WVW = WV * W;
+  const Eigen::Matrix<Scalar, 3, 3> VWW = VW * W;
+  const Eigen::Matrix<Scalar, 3, 3> WWV = - VWW.transpose();    // Note on this change wrt. Barfoot: it happens that V*W*W = -(W*W*V).transpose() !!!
```

Comment:
Do we really need this temporary ? It is used only once.
Same remark in Jl.

Reply:
Which one, WWV? No, it could go straight into the formula. Change it if you wish, but keep the comment otherwise we'll  lose track.

Reply:
OK I do it

Reply:
done and pushed